src/Sdk/DTWebApi/WebApi/RunnerRefreshMessage.cs

```diff
@@ -15,35 +16,32 @@ internal RunnerRefreshMessage()
         {
         }
 
-        public RunnerRefreshMessage(
```

Comment:
just to double check, RunnerRefreshMessage is not currently used in any production environment, so changing the contract won't cause any compatibility problem.

Reply:
correct